docs/source/getting-started/docker.rst

```diff
@@ -0,0 +1,59 @@
+==================================
+Set Up a Spark Cluster with Docker
+==================================
+
+This page describes how to set up a Spark cluster locally on your machine by using Docker containers. This approach is useful if you do not need a high-level of performance, and want to quickly try out the Migrator without having to set up a real cluster of nodes. It requires Docker and Git.
+
+1. Clone the Migrator repository.
+
+   .. code-block:: bash
+
+     git clone https://github.com/scylladb/scylla-migrator.git
+     cd scylla-migrator
+
+2. Download the latest release of the ``scylla-migrator-assembly.jar`` and put it in the directory ``migrator/target/scala-2.13/``.
```

Comment:
why migrator/target/scala... ?
let's remove this, you removed it in other scripts too, let's abstract from scala version in path and just mark it as requirement from spark version

Reply:
Agreed. The Ansible playbook now downloads the .jar and puts it in the `scylla-migrator/` root directory.